crates/rvm-rs/src/constants.rs

```diff
@@ -0,0 +1,44 @@
+use semver::Version;
+
+use crate::errors::Error;
+
+/// Repository URL for list.json files with supported Resolc versions
+pub const REPO_URL: &str =
```

Comment:
is svm downloading compilers directly from repo or GH pages?

Reply:
now using github pages, svm downloads from `binaries.soliditylang.org`